src/helpers/__tests__/utils.test.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, it, expect } from 'vitest';
+
+import { createErc1155Metadata } from '../utils';
+
+describe('utils', () => {
+  describe('createErc1155Metadata', () => {
+    it('should create metadata', () => {
+      const metadata = createErc1155Metadata({
+        name: 'Test',
+        description: 'Test description',
+        externalUrl: 'https://test.com',
+        githubUrl: 'https://github.com',
+        twitterUrl: 'https://twitter.com',
+        discordUrl: 'https://discord.com',
```

Comment:
We should not use URLs but usernames instead
```suggestion
        github: 'snapshot-labs',
        twitter: 'SnapshotLabs',
        discord: 'snapshot',
```